src/main/java/is/ru/tictactoe/webui/WebUI.java

```diff
@@ -21,6 +21,7 @@
     private static final String GAMETYPE = "unknown";
     private static Player player1;
     private static Player player2;
+    private static int numberOfGames = 0;
```

Comment:
This must have slipped review last time.

Use of static variables here will probably not do what you want, as concurrent clients will update them independently.  There is only *one* instance of them for the entire web server.

We want to store all of this in the session.